misc/deployments/test4.gno.land/genesis.json

```diff
@@ -1474,11 +1474,11 @@
               "files": [
```

Comment:
this file won't need to be patched, but it's actually not a big problem to do it.